core/src/main/java/dk/alexandra/fresco/lib/real/RealLinearAlgebra.java

```diff
@@ -26,6 +26,33 @@
    */
   DRes<Matrix<DRes<SReal>>> add(Matrix<BigDecimal> a, DRes<Matrix<DRes<SReal>>> b);
 
+  /**
+   * Subtracts two secret values and returns the result.
+   *
+   * @param a Secret value 1
```

Comment:
This is a bit confusing, the value does not have to do with the number 1. How about `first secret value`/`second secret value`